content/blog/2020/01/2020-01-25-jenkins-world-lisbon-with-love-from-india.adoc

```diff
@@ -0,0 +1,105 @@
+---
+layout: post
+title: "Jenkins World Lisbon Experience"
+tags:
+- gitlab
+- lisbon
+- devopsworld
+- jenkinsworld
+- gsoc
+- gsoc2019
+author: baymac
+description: "Experience of attending Jenkins World Lisbon 2019"
+opengraph:
+  image: /images/post-images/2020/gsoc-jenkins-world-parichay/project-presentation.jpg
+---
+
+= My personal experience of Jenkins World Lisbon
+
+After an amazing three months of development period in the summers of 2019 with Jenkins Project, I was a better developer, loved open source, met
+passionate people and had fun at work. Jenkins is not just a community, it is a family. When GSoC period was over, we received swags from Jenkins. 
+Natasha Stopa (one of the GSoC 19 student) was invited to attend Jenkins World San Fransisco. It was nice to see her enjoy there. But guess what? Jenkins 
+also invited three other students (Abhyudaya, Long and me) to Jenkins World Lisbon. I was super psyched when Marky Jackson (one of my project mentor) 
+broke the news to me. 
+
+The trip to Lisbon required to sort a few things like flight tickets, hotel booking, passport, visa etc. Oleg Nenashev had scheduled meetings to discuss
+and help us with arranging everything for our travel. Thanks to him. :)
+
+== From India to Lisbon
+
+Abhyudaya and I boarded our flight from Indira Gandhi Airport (New Delhi) to Lisbon on December 1, 2019 morning at 0500 hours (local time). It was a fine
+trip with an hour layover in Istanbul Ataturk Airport. We arrived in Lisbon at 1500 hours (local time). The weather in Lisbon was terrific. A mild cold
+but strong sea breeze was the starting point of me falling in love with the place. We arrived at our hotel (Novotel Lisboa) in an Uber. Oleg met us at 
+the lobby to help us with check-in. It was great to finally meet him in person after months of knowing and working together. We had a good chat about the 
+event, what to expect and other sightseeing areas. After a short time of refresh, Long who travelled from Berlin a day before met us at the restaurant. We 
+had brief chat knowing each other, had our food and went to bed.
+
+== Day 0
+
+The next day (Dec 2) was Hackfest. The event managing team had an app for DevOps World | Jenkins World (DWJW) Lisbon with all schedules and other informations
+which was incredibly convenient for all attendees. We hit the bed early so that we could wake up early to have our breakfast and catch the shuttle to Centro 
+de Congressos de Lisboa (CCL) where the event was organised. I woke up early for a short jog in the streets. Lisbon is a city made on hills. The streets have 
+beautiful mosaic styled pavements. It was nice to see around the city. Then Abhyudaya and me went for breakfast and reached CCL in an Uber at 0815 hours.
+
+image::/images/post-images/2020/gsoc-jenkins-world-parichay/with-oleg.jpg[A picture with Oleg Nenashev]
```

Comment:
@baymac  File name extension for picture is wrong. You have lowercase but the uploaded file is uppercase.